pkg/network/ovn_kubernetes.go

```diff
@@ -127,6 +129,17 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		nb_inactivity_probe = "60000"
 		klog.Infof("OVN_NB_INACTIVITY_PROBE env var is not defined. Using: %s", nb_inactivity_probe)
 	}
+
+	// Hypershift
+	data.Data["ManagementClusterName"] = os.Getenv("MANAGEMENT_CLUSTER_NAME")
+	data.Data["ManagementClusterDomain"] = os.Getenv("MANAGEMENT_CLUSTER_DOMAIN")
+	data.Data["HostedClusterNamespace"] = os.Getenv("HOSTED_CLUSTER_NAMESPACE")
+	data.Data["OvnkubeMasterReplicas"] = len(bootstrapResult.OVN.MasterIPs)
+
+	// sbdb route in the format of "ssl://ovnkube-sbdb-<hostedcluster namespace>.apps.<hostedcluster domain name>:443"
+	data.Data["OVN_NB_DB_ROUTE"] = fmt.Sprintf("ssl://ovnkube-sbdb-%s.apps.%s:%s", os.Getenv("HOSTED_CLUSTER_NAMESPACE"), os.Getenv("MANAGEMENT_CLUSTER_DOMAIN"), OVN_SB_ROUTE_PORT)
```

Comment:
TODO: get this from the route Status.Ultimately that logic will need to be in Bootstrap probably.

Reply:
Added TODO comment.